db/migrate/20160831015641_remove_has_muted_from_users.rb

```diff
@@ -0,0 +1,8 @@
+class RemoveHasMutedFromUsers < ActiveRecord::Migration
+  def change
+    User.where(has_muted: true).each do |user|
+      user.experienced!("threadMuted")
```

Comment:
mutedThread above vs threadMuted here. I think that's a mistake

I think the best name is the threadMuted one.. because it puts the most important/categorising word first..